templates/books/book_detail.html

```diff
@@ -31,14 +31,14 @@ <h2>{{ book.title }}</h2>
 <form method="post" class="book__form" action="">
 {% csrf_token %}
 
+<input type="hidden" name="status" value="{{ book.status }}" id="id_status">
```

Comment:
Do you really need this  field on the form anymore? I think you may get away without it :)

Reply:
I just tried, it doesn't work without the field - it seems the form code is ignored then; I think at least one field is needed to make it count as a form?

Reply:
But it is not needed for the logic. Hopefully I can get rid of it when I rewrite things.